### PR TITLE
Add my works and about me sections in the mobile version of portfolio project

### PR DESCRIPTION
**Pull request by Stanleeeeee**
This is the mobile version of the portfolio project. 
- The about me section,
- My work section and
- some project sections were added.

Please review my project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,6 +60,175 @@ <h3 class="H-title">I'am a Software Developer</h3>
...
+                        ever since the 1500s, when an unknown printer took a standard
+                        dummy text. A daily selection of privately personalized reads; no
+                        accounts or sign-ups required.
+                      </p>
+                      <div class="language-container" data-language="language-list">
+                        <div class="box"><span>css</span></div>
+                        <div class="box"><span>html</span></div>
+                        <div class="box"><span>bootstrap</span></div>
+                        <div class="box"><span>Ruby</span></div>
+                      </div>
```
Comment (CollinsTatang) [inline index.html]: - [x] Please wrap the technology in a `ul` tag and each should be in a `li tag`.

File: style.css [hunk 2]
```diff
@@ -130,3 +130,267 @@ p {
...
+  text-align: center;
+  font-size: 72px;
+  font-family: "Crete Round", sans-serif;
+  color: #ff6b00;
+  line-height: 96%;
+  letter-spacing: 0.37px;
+  vertical-align: top;
+  width: 387px;
+  padding-bottom: 50px;
+}
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please work on your entire style following the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
### ### **Take a look at your design and mobile view**
mobile

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Stanleeeeee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (Stanleeeeee) [reply style.css]: Please can you explain more about this

Comment (CollinsTatang) [reply style.css]: Hi @Stanleeeeee 
Please watch the short [video](https://loom.com/share/00c6d4da96664c97810c33316f292094) and see how the project is displayed on mobile

**Review by Stanleeeeee — COMMENTED** [review]
Added some corrections on the portfolio mobile viewport and also corrected the div tag to ul tag.
Please review my project

File: index.html [hunk 3]
```diff
@@ -60,6 +61,189 @@ <h3 class="H-title">I'am a Software Developer</h3>
...
+                        dummy text. A daily selection of privately personalized reads; no
+                        accounts or sign-ups required.
+                      </p>
+                      <div>
+                      <ul class="language-button12">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                      </ul>
```
Comment (OLIPLICHE) [inline index.html:99]: - [ ] Nice Job for implementing the change👏 👏.

File: index.html [hunk 4]
```diff
@@ -60,6 +61,189 @@ <h3 class="H-title">I'am a Software Developer</h3>
...
+                        See Project
+                      </button>
+                    </div>
+                    </div>
+                  </div>
+                </div>
+              </section>
+
+              <!--Some work section-->
+              <section id="container-grid">
```
Comment (OLIPLICHE) [inline index.html:114]: - [ ] Please you did not solve what the previous review pointed out. Kindly open this  [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278) link and follow the design. 
- [ ] I provided the pictures of your design below to bring more explanations on the video from the previous review. 
- Kindly focus on the red lines I drawn and compare your design to the original  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278)
- Look
WhatsApp Image 2022-03-26 at 12 34 01 PM
- Look
WhatsApp Image 2022-03-26 at 12 34 01 PM (1)
- Look
WhatsApp Image 2022-03-26 at 12 34 01 PM (2)

- Look
WhatsApp Image 2022-03-26 at 12 34 01 PM (3)

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Stanleeeeee, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
 ## Highlights
✔️ Linter checks pass
✔️ Descriptive README
✔️ Descriptive PR

 ## Required Changes ♻️
 

- [ ] CHECK THE COMMENTS UNDER THE REVIEW

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] **You can consider this**: Please follow the Design provided so that you will build a nice Portfolio
- The picture below is from Figma Design

t

- This one is From your design: 
WhatsApp Image 2022-03-26 at 12 45 37 PM





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Stanleeeeee) [reply index.html:114]: @OLIPLICHE , what exactly am i expected to do in this section. I still don't understand what the issue is. I have used the figma design for mobile scaling. What exactly am I expected to do

Comment (Stanleeeeee) [reply index.html:114]: We have not been taught anything about using the inspect property. This is my second week at microverse. I'm not having the above issues when I minimize my browser to mobile version

**Review by Stanleeeeee — COMMENTED** [review]
Added some corrections to the work section

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Stanleeeeee, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**